modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_localized.js

```diff
@@ -659,6 +659,26 @@ AUI.add(
 
 					instance._updateTranslationStatus(selectedLanguageId);
 				},
+
+				updateInputPlaceholder(editor) {
+					var instance = this;
+
+					var inputPlaceholder = instance.get(STR_INPUT_PLACEHOLDER);
+
+					var nativeEditor = editor.getNativeEditor();
+
+					var newPlaceholder = A.one(
+						'#' + nativeEditor.element.getId()
+					);
+
+					if (inputPlaceholder.compareTo(newPlaceholder)) {
+						return;
+					}
+
+					instance.set('editor', window[nativeEditor.name]);
```

Comment:
Isn't `editor === `window[nativeEditor.name]` ?

Reply:
Yes, you're right that's not needed